bookwyrm/templates/layout.html

```diff
@@ -8,24 +8,25 @@
     <link rel="stylesheet" href="/static/css/vendor/icons.css">
     <link rel="stylesheet" href="/static/css/bookwyrm.css">
 
-    <link rel="shortcut icon" type="image/x-icon" href="{% if site.favicon %}/images/{{ site.favicon }}{% else %}/static/images/favicon.ico{% endif %}">
+    <link rel="shortcut icon" type="image/x-icon" href="{% if site.favicon %}{{ media_url }}{{ site.favicon }}{% else %}/static/images/favicon.ico{% endif %}">
 
     <meta name="twitter:card" content="summary">
```

Comment:
It depends on the goal to be achieved. I personally find those banners too invasive and would be content with a summary. I read a large image as some text in all caps; it screams at me.

Reply:
Fun fact: [Twitter documentation has been needing fixing for 3.5 years](https://stackoverflow.com/questions/47578130/twitter-card-summary-vs-summary-large-image#47580931) :)

Reply:
for this image format, I think `summary_large_image` makes sense. It looks like mastodon still uses the thumbnail-on-the-left style regardless

Reply:
Not to say you or anybody is wrong or right, but why do you think it makes sense? Maybe I’m missing something obvious.

- What do you mean by “for this image format”?
- What is the goal of having the large format over the smaller one?

Reply:
I mean the images that are generated are formatted such that they look super nice as big twitter or facebook cards:
<img width="555" alt="Screen Shot 2021-06-18 at 9 44 56 AM" src="https://user-images.githubusercontent.com/1807695/122593054-f221e700-d019-11eb-9ed6-6ebb48df737d.png">

I think the result looks great!

Reply:
Right now the images are optimized for horizontal display, not for square display. If the choice is to display the images as `summary` (127x127 on the Twitter interface, 60x60 in the Mastodon feed), there will be the problem of legibility.

When I first chose this format, it was for a `summary_large_image`-type display on Twitter, and so that image could be passed as an illustration on Fediverse posts:

![image](https://user-images.githubusercontent.com/1781070/122594052-cf69ef80-d066-11eb-825d-9fda78113ef3.png)
 
This PR doesn't transmit the preview image in the posts, because it's not the scope (yet). The idea would be to send the URL of the book cover and of the book preview, so that mainstream Fediverse instances would display the book preview as main image, and BookWyrm would display the book cover.

Reply:
Here's how it could look like on the Mastodon feed

![baru-generated](https://user-images.githubusercontent.com/1781070/122595866-5b7d1680-d069-11eb-810d-0762f134f303.jpg)

Reply:
> I mean the images that are generated are formatted such that they look super nice as big twitter or facebook cards:

Ha! I thought those were more like screenshots than plain images. I can see clearly now… :notes:

I’m going back to my cave; sorry for the noise. :)